readme.md

```diff
@@ -34,16 +34,16 @@ gulp.task('default', function () {
 
 ##### ui
 
-Type: `string`  
```

Comment:
apologies, editor did this automatically.  I'm curious, what's the purpose here?